sample-queries/sample-queries.json

```diff
@@ -1679,6 +1679,65 @@
         "tip": "enable ExternalItem.Read.All",
         "skipTest": false
     },
+    {
+        "id": "26743d2a-2306-40f9-b3e8-3f857af69269",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List ediscoveryCases",
+        "requestUrl": "/beta/compliance/ediscovery/cases",
+        "docLink": "https://docs.microsoft.com/graph/api/ediscoverycase-list?view=graph-rest-beta",
+        "skipTest": false
+    },
+    {
+        "id": "d3f13cd9-2c89-4876-a88c-081e41370a02",
+        "category": "Compliance (beta)",
+        "method": "POST",
+        "humanName": "Create ediscoveryCase",
+        "requestUrl": "/beta/compliance/ediscovery/cases",
+        "headers": [
+            {
+                "displayName": "My first eDiscovery case"
+            }
+        ],
+        "docLink": "https://docs.microsoft.com/graph/api/ediscoverycase-post?view=graph-rest-beta",
+        "skipTest": false
+    },
+    {
+        "id": "9caaaa95-4f2b-4404-83f1-990fea9a3fc5",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List reviewSets",
+        "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}/reviewSets",
+        "docLink": "https://docs.microsoft.com/graph/api/reviewset-list?view=graph-rest-beta",
+        "tip": "Replace {caseId} with the ID from an eDiscoveryCase (List eDiscovery Cases)",
+        "skipTest": false
+    },
+    {
+        "id": "f9b82da2-91ca-4e87-8ac7-39d8c755050d",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List reviewSetQueries",
+        "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}/reviewSets/{reviewSetId}/queries",
+        "docLink": "https://docs.microsoft.com/graph/api/reviewsetquery-list?view=graph-rest-beta",
+        "tip": "Replace {caseId} with the ID from an eDiscovery Case (List eDiscovery Cases) and {reviewSetId} with the ID from a review set that exists in that case (List reviewSets)",
+        "skipTest": false
+    },
+    {
+        "id": "af8a5db2-d4c6-47b9-a62f-5cfdbf159aa2",
+        "category": "Compliance (beta)",
+        "method": "POST",
+        "humanName": "Create reviewSetQuery",
+        "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}/reviewSets/{reviewSetId}/queries",
+        "headers": [
+            {
+                "displayName":"Quarterly Financials Query",
+                "query":"(subject:\"Quarterly Financials\")"
+            }
+        ],
+        "docLink": "https://docs.microsoft.com/graph/api/reviewsetquery-post?view=graph-rest-beta",
+        "tip": "Replace {caseId} with the ID from an eDiscovery Case (List eDiscovery Cases) and {reviewSetId} with the ID from a review set that exists in that case (List reviewSets)",
```

Comment:
Give a query example for getting one of the id on the tip message so the user knows.

Reply:
Can you point me to an example of what you mean?  There is a tip there and it references back to the APIs to use to get those on line 1738.  There's a bit of a conundrum here because getting the second ID requires the first ID. The tip would be a help topic by itself.

Reply:
Fixed in all calls that have CaseID.  I appended the following text which I stole from another sample...

 To find the ID of an eDiscovery case, you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases.

There are some samples that also require another ID, I've left that out, hopefully they can figure it out once they see the patter.  If there's a more succinct way to express looking up and id and then using that to look up another ID, I'm all ears.

Reply:
Sorry this wasnt clear, my suggestion is have the tip message as
Replace {caseId} with the ID from an eDiscovery Case (https://graph.microsoft.com/beta/compliance/ediscovery/cases) and {reviewSetId} with the ID from a review set that exists in that case (https://graph.microsoft.com/beta/compliance/ediscovery/cases/{caseid}/reviewSets)

Reply:
Gotcha, thanks for the clear example.  Incorporated in all examples where it makes sense.